tests/MongoDB.Driver.Core.Tests/Core/Operations/CreateCollectionOperationTests.cs

```diff
@@ -470,7 +470,7 @@ public void CreateCommand_should_throw_when_Collation_is_set_and_not_supported()
             [Values(false, true)]
             bool async)
         {
-            RequireServer.Check();
+            RequireServer.Check().VersionLessThan("3.7.0");
```

Comment:
Nit: No need to mention CSHARP-2221 in the commit log for this.  It's unrelated.